skyportal/handlers/api/source.py

```diff
@@ -291,6 +292,25 @@ def get(self, obj_id=None):
                 )
                 .all()
             )
+            # add the date(s) this source was saved to each of these groups
+            new_group_list = []
+            for g in source_info["groups"]:
+                saved_at = [
```

Comment:
Shouldn't there be only one saved_at value for each GroupSource row?

Reply:
Probably should be a scalar value for each group rather than a list

Reply:
Also, query.all() already returns a list -- here you're wrapping it in another list

Reply:
I think you can scrap the list comprehension and use `.first()`

Reply:
I think we talked about having the ability to save the same source to the same group more than once. 

In any case, it doesn't hurt to have it as an array if we ever do decide that is an option.

Reply:
That's not even possible with the current data model. There can only be one GroupSource row for a given obj_id, group_id pair, and saved_at is necessarily a single scalar value. Please update this to reflect the current data model. If we ever refactor that in the future, we can update this accordingly. Thanks

Reply:
OK